Description: Fix data path for deb package build
Author: Fabio Fantoni <fabio.fantoni@tiscali.it>
--- a/src/Teambuilder/pluginmanager.cpp
+++ b/src/Teambuilder/pluginmanager.cpp
@@ -265,7 +265,11 @@
     QFileDialog *fd = new QFileDialog(this);
     fd->setAttribute(Qt::WA_DeleteOnClose, true);
     fd->setFileMode(QFileDialog::ExistingFile);
+#if defined(DEB_PKG)
+    fd->setDirectory("/usr/lib/pokemon-online/clientplugins");
+#else
     fd->setDirectory("myplugins");
+#endif
     fd->show();
 
     connect(fd, SIGNAL(fileSelected(QString)), this, SLOT(addPlugin(QString)));
--- a/src/Server/Server.pro
+++ b/src/Server/Server.pro
@@ -5,7 +5,8 @@
     script \
     xml \
     sql
-TARGET = Server
+CONFIG(debian_package): TARGET = Pokemon-Online-Server
+!CONFIG(debian_package): TARGET = Server
 DESTDIR = ../../bin
 TEMPLATE = app
 SOURCES += main.cpp \
--- a/src/Server/pluginmanager.cpp
+++ b/src/Server/pluginmanager.cpp
@@ -209,7 +209,11 @@
     QFileDialog *fd = new QFileDialog(this);
     fd->setAttribute(Qt::WA_DeleteOnClose, true);
     fd->setFileMode(QFileDialog::ExistingFile);
+#if defined(DEB_PKG)
+    fd->setDirectory("/usr/lib/pokemon-online/serverplugins");
+#else
     fd->setDirectory("serverplugins");
+#endif
     fd->show();
 
     connect(fd, SIGNAL(fileSelected(QString)), this, SLOT(addPlugin(QString)));
